### ISSUE TITLE
undeclared identifier: 'customAsyncIterator'

### ISSUE THREAD

**Issue by glyh** [SirNickolas/asyncIters-Nim#1]
Hello, I'm trying out this package and I got this: 
```
/home/lyh/.nimble/pkgs2/asyncIters-1.2.4-708d1f4ad4e8fc7843b479806c5ffec1b239b81e/asyncIters.nim(68, 28) Error: undeclared identifier: 'customAsyncIterator'
candidates (edit distance, scope distance); see '--spellSuggest': 
 (0, 4): 'customAsyncIterator'
nimble.nim(229)          buildFromDir

    Error:  Build failed for the package: jmetav
```

I'm on asynciters >= 1.2.4

**Comment by glyh** [comment]
Looks like this can only be reproduced with
```nim
from asyncIters import AsyncIterator
```

**Comment by SirNickolas** [comment]
Congratulations; you’ve found a [bug in the compiler][bug]!

Getting a compilation error after merely importing a library is indeed frustrating. I’ve fixed the *undeclared-identifier* error in v1.2.5. Please note that [v1.3.0][] is also available.

I should, however, point out that in order to do something useful, you need much more in scope than just `AsyncIterator`. What’s worse, the exact list of required symbols varies between stdlib’s versions (and is completely different in Chronos). Hence it is safer to just import everything from `asyncIters`; it [attempts][exports] not to pollute the namespace too much.

[bug]: https://github.com/nim-lang/Nim/issues/23406
[v1.3.0]: https://github.com/SirNickolas/asyncIters-Nim/releases/tag/v1.3.0
[exports]: https://github.com/SirNickolas/asyncIters-Nim/blob/v1.3.0/src/asyncIters.nim#L33
